### DISCUSSION TITLE
posts/2023/8d14a4cf/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#17] [Announcements]
# posts/2023/8d14a4cf/

去他妈的生活，去他妈的苟且。

https://blog.gzti.me/posts/2023/8d14a4cf/

**Comment by yemq3** [comment]
给点鸡汤,如果把历史拉开来看的话,历史上绝大多数的人都处在生产力发展缓慢的时代,工业革命之后就是两次大战,新中国在改革开放后经济才飞速发展,在我们这代人之前,其实也没多少人能生逢盛世平安过一辈子.就业/考研/润,其实不管怎么选,你大概率都能过得比大多数人好,别让自己后悔就行.有空来深圳约饭😀

**Comment by hmt38** [comment]
Take it easy ! my bro

**Reply by hmt38** [reply]
thai is always there for you...

**Comment by Cyy0808** [comment]
Marxismus欢迎你

**Comment by charlie0129** [comment]
太赞同了，像极了我的心路历程

**Comment by KlariZhang** [comment]
hello陌生人，我现是个还不会写notebook网站的即将升入大二的数学和计算机专业的学生，上个学期我给自己定下的想挑战自我的目标把自己卷到中度抑郁，我讨厌上海，恨死那里的窒息感，每周只能靠着上海“新店开业”吃点新鲜玩意，想着内卷的大家和自己，不断地被驯化，没有了争取自己的空间的勇气，破烂的宿舍和极其高压的环境，我绝望的呼吸不上来，或许意气风发应该褪去，只渴望一段平淡的亲密关系和稳定的知识和收入，原来人生，就是这样。希望下次可以来你的博客速通学习我的专业课，也祝你毕业快乐，人生顺利

**Reply by KlariZhang** [reply]
生活磨平了我的热情，但我希望在我的学生时代，我可以过的潇洒自在些，即使没有那些浮光掠影，只需要，独善其身。生命的意义在于成为自己不是吗，愿大家都好，带着100%的对数学和逻辑的热情，我忘记了我学习的初心，是为了探索世界的不确定性

**Reply by GZTimeWalker** [reply]
你好哦！如果你想学习专业课这里建议你去：

- https://note.tonycrane.cc/
- https://isshikihugh.github.io/zju-cs-asio/

**Reply by KlariZhang** [reply]
啊啊啊，这个博客竟然是活的，好的好的，谢谢！

**Reply by GZTimeWalker** [reply]
在这里发评论，我会收到 GitHub 发送的通知的，所以会比较及时的回复（

**Reply by KlariZhang** [reply]
soga！感觉github好高级！我刚在csdn上学习使用就无意逛到你的博客了哈哈哈哈，祝你学业和生活都顺利呀，即便人生平淡人生也不会完蛋嘛，一遍去他妈的生活一边好好生活呀！感觉找到宝藏啦，接下来的专业课有buff加持了哈哈哈哈哈

**Reply by GZTimeWalker** [reply]
谢谢你！祝你一切顺利！
